bin/development-server.js

```diff
@@ -76,3 +76,9 @@ app.listen(8000, "0.0.0.0", function(err, result) {
 
   console.log("Development Server Listening at http://localhost:8000");
 });
+
+const examples = express();
+examples.use(express.static("public/js/test/examples"));
+examples.listen(7999, "0.0.0.0", function(err, result) {
+  console.log("View debugger examples at http://localhost:7999");
```

Comment:
We already have a bunch of messages and it's starting to feel like spam... do we need this message?